rrule/src/core/rruleset.rs

```diff
@@ -35,9 +37,24 @@ impl RRuleSet {
             rdate: vec![],
             exrule: vec![],
             exdate: vec![],
+            limit: Some(u16::MAX),
         }
     }
 
+    /// Disables validation limits
+    #[must_use]
+    pub fn disable_validation_limits(mut self) -> Self {
+        self.limit = None;
+        self
+    }
+
+    /// Set the validation limit
+    #[must_use]
+    pub fn set_limit(mut self, limit: u16) -> Self {
```

Comment:
What about to name it `limit`, instead of `set_limit`?

Reply:
Agreed, that would be more consistent with the other method names 👍